src/ai/backend/manager/scheduler/dispatcher.py

```diff
@@ -293,16 +293,9 @@ def __init__(
         self.registry = registry
         self.lock_factory = lock_factory
         self.db = registry.db
-        redis_profile_target: RedisProfileTarget = RedisProfileTarget.from_dict(
-            self.config_provider.config.redis.model_dump()
-        )
         # redis_live will be initialized in __ainit__
         self.redis_live = cast(ValkeyLiveClient, None)
-        self.redis_stat = redis_helper.get_redis_object(
-            redis_profile_target.profile_target(RedisRole.STATISTICS),
-            name="stat",
-            db=REDIS_STATISTICS_DB,
-        )
+        self.valkey_stat_client = cast(ValkeyStatClient, None)
```

Comment:
I know this is because `ValkeyStatClient` requires asynchronous task to initialize
Let's go with this for now but we should change such codes